dockerfiles/submitty/tutorial/database_client/Dockerfile

```diff
@@ -199,6 +199,9 @@ RUN set -eux; \
 	pip --version
 
 # Install additional packages via pip
+RUN ln -s /usr/local/bin/python3 /usr/bin/python3 \
```

Comment:
would it be better to add /usr/local/bin to the path?  (would that solve the problem?)

Reply:
The tutorial wants /usr/bin/python3 to be the location for python3 for some reason. The path already has /usr/local/bin.

Reply:
Sorry I got that backwards
Is /usr/bin on the path?

Reply:
I'd rather standardize the tutorial than hack a symlink

Reply:
![image](https://github.com/Submitty/DockerImages/assets/46644513/fa857a34-a4c4-4ce2-8a65-5f0089832c0e)
This is the PATH inside docker container tutorial:db_client